### DISCUSSION TITLE
Are there constants for `UTIME_NOW` or `UTIME_OMIT`?

### DISCUSSION THREAD

**Discussion by overheadhunter** [winfsp/winfsp#445] [Q&A] [answered]
When using the FUSE API to implement a file system, one needs to respect the magic numbers `UTIME_NOW` and `UTIME_OMIT`. The value is OS-dependent, but I couldn't find such values on Windows. Does WinFSP define them somewhere or is this irrelevant, since there are no `utimens` calls on Windows that make use of these values? (Not even on WSL?)

**Comment by billziss-gh** [comment, marked as answer]
WinFsp-FUSE will never send you these values. It will always send real timestamps.

Relevant code: https://github.com/winfsp/winfsp/blob/master/src/dll/fuse/fuse_intf.c#L1577-L1610
